lib/ecto/adapters/sql.ex

```diff
@@ -38,19 +38,15 @@ defmodule Ecto.Adapters.SQL do
       ## Types
 
       @doc false
-      def autogenerate(:id), do: nil
-      def autogenerate(:embed_id), do: Ecto.UUID.autogenerate()
-      def autogenerate(:binary_id), do: Ecto.UUID.autogenerate()
+      def autogenerate(:id),        do: nil
+      def autogenerate(:embed_id),  do: Ecto.UUID.generate()
```

Comment:
@michalmuskala is it safe to say that you never see the `{{..., ..., ...}, {..., ..., ...}}` for datetimes in the Mongo adapter?